plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -97,8 +97,8 @@ SecRule REQUEST_FILENAME "@endsWith /wp-comments-post.php" \
     ver:'wordpress-rule-exclusions-plugin/1.0.1'"
 
 SecRule REQUEST_FILENAME "@endsWith /wp-admin/comment.php" \
-    "id:9507170,\
-    phase:2,\
+    "id:9507131,\
+    phase:1,\
```

Comment:
I don't know if this is something that should be considered a breaking change, I mean what happens if somebody is disabling this rule via a ctl:action for whatever reason?

Reply:
Renumbering an exclusion rule is usually not considered as a breaking change. I don't think anybody is excluding an exclusion rule.